MAUI/TextInputLayout/Custom-Font.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Font Customization  in .NET MAUI TextInputLayout control | Syncfusion
+description: Learn here all about Font Customization  support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Font Customization  in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+You can customize the appearance (size, attributes, and family) of font by setting the FontFamily, FontSize, and FontAttributes properties of the LabelStyle property.
+
+Refer to this [documentation](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/fonts) to configure the custom fonts in .NET MAUI.
+
+## Hint
+
+You can customize the font of `hint` label by setting the FontFamily, FontSize, and FontAttributes, properties of HintLabelStyle in SfTextInputLayout.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<inputLayout:SfTextInputLayout Hint="Name"
+                               ContainerType="Outlined"
+                               HelperText="Enter your name">
+    <Entry />
+    <inputLayout:SfTextInputLayout.HintLabelStyle>
+        <inputLayout:LabelStyle FontSize="16" FontFamily="Lobster-Regular"/>
+    </inputLayout:SfTextInputLayout.HintLabelStyle>
+</inputLayout:SfTextInputLayout> 
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.ContainerType = ContainerType.Outlined;
+inputLayout.HelperText = "Enter your name";
+inputLayout.HintLabelStyle = new LabelStyle() { FontFamily = "Lobster-Regular", FontSize = 16};
+inputLayout.Content = new Entry(); 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Hint label style](images/CustomFont/HintLabelStyle.png)
+
+## Helper text
+
+You can customize the font of `helper text` label by setting the FontFamily, FontSize, and FontAttributes properties of HelperLabelStyle in SfTextInputLayout.
```

Comment:
You can customize the font of the `helper text` label by setting the FontFamily, FontSize, and FontAttributes properties of HelperLabelStyle in the SfTextInputLayout.

Reply:
Addressed the changes